CMakeLists.txt

```
@@ -4,6 +4,13 @@ project(gz_ogre_next_vendor)
 find_package(ament_cmake REQUIRED)
 find_package(ament_cmake_vendor_package REQUIRED)
 
+set(EXTRA_CMAKE_FLAGS)
+if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm64|aarch64|armhf")
```

```suggestion
if(CMAKE_SYSTEM_PROCESSOR MATCHES "^(aarch64.*|AARCH64.*|arm64.*|ARM64.*|arm.*|ARM.*)")
```

Looking to include the Windows options just in case, I just copy/merged the values used by OpenCV https://github.com/opencv/opencv/blob/ba65d2eb0d83e6c9567a2534f8994cd4484a1635/cmake/OpenCVDetectCXXCompiler.cmake#L94